references/cli/def.go

```diff
@@ -325,6 +327,40 @@ func NewDefinitionGetCommand(c common.Args) *cobra.Command {
 	return cmd
 }
 
+// NewDefinitionGenDocCommand create the `vela def gen-doc` command to generate documentation of definitions
+func NewDefinitionGenDocCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "gen-doc NAME",
+		Short: "Generate documentation of definitions (Only Terraform typed definitions are supported)",
```

Comment:
can we also support other cue based definition?

Reply:
Yes, it's in my plan. This feature is the official feature by CLI to generate docs for KubeVela definitions (this was supported by https://github.com/oam-dev/kubevela/blob/master/hack/references/generate.go). I tried to generate all definitions (including workflowstep), but 1) there were some blockers to run the tool, 2) the generated docs differs from thoes hosted in kubevela.io. So I plan to support Terraform typed definition first, and then for all components, traits, policy and workflow. Is that okay?

Reply:
sure,thanks